docs/ABOUT.md

```diff
@@ -1,7 +1,9 @@
-Elm's goal is to be the best of functional programming in the browser.
+Elm is a pure, functional language that compiles to javascript.
 
-One of the primary goals of Elm is to completely eliminate runtime exceptions before they reach production by combining an aggressively helpful compiler with solid architectural principles.
+It was designed by Evan Czaplicki as part of writing his thesis on [Concurrent FRP for Functional GUIs](http://elm-lang.org/papers/concurrent-frp.pdf) and evolved into a language to help make making user interfaces better.
 
-Elm was designed by Evan Czaplicki as part of writing his thesis on [Concurrent FRP for Functional GUIs](http://elm-lang.org/papers/concurrent-frp.pdf).
+Elm's goal is to be the simplest and best language to use to make web-based user interfaces.
+
+One of the primary goals of Elm is to completely elminate runtime exceptions before they reach production by combining an aggressively helpful compiler with solid architectural principles.
```

Comment:
It's a little odd to have "Elm's goal" above and then "Elm's goals" here. Some learners (who are perhaps new to coding or don't come from a software engineering background) might also be unfamiliar with the colloquial use of "development" and "production" as environments.

I might change this sentence to 

> Elm aims to completely eliminate runtime exceptions by combining an aggressively helpful compiler with solid architectural principles.